Mathlib/AlgebraicGeometry/Morphisms/Constructors.lean

```diff
@@ -240,17 +261,15 @@ lemma topologically_propertyIsLocalAtTarget
     (hP₃ : ∀ {α β : Type u} [TopologicalSpace α] [TopologicalSpace β] (f : α → β) {ι : Type u}
       (U : ι → TopologicalSpace.Opens β) (_ : iSup U = ⊤) (_ : Continuous f),
       (∀ i, P ((U i).carrier.restrictPreimage f)) → P f) :
-    PropertyIsLocalAtTarget (MorphismProperty.topologically P) := by
-  apply propertyIsLocalAtTarget_of_morphismRestrict
+    IsLocalAtTarget (MorphismProperty.topologically P) := by
+  apply IsLocalAtTarget.mk'
   · intro X Y f U hf
     simp_rw [MorphismProperty.topologically, morphismRestrict_val_base]
     exact hP₂ f.val.base U.carrier hf
   · intro X Y f ι U hU hf
-    apply hP₃ f.val.base U hU
-    · exact f.val.base.continuous
-    · intro i
-      rw [← morphismRestrict_val_base]
-      exact hf i
+    apply hP₃ f.val.base U hU f.val.base.continuous fun i ↦ ?_
+    rw [← morphismRestrict_val_base]
+    exact hf i
```

Comment:
I can't comment on unchanged code, but I think that the name of this lemma should be changed to `topologically_isLocalAtTarget`.